docker/DOCKER.md

```diff
@@ -0,0 +1,51 @@
+
+
+# Run Waltz
+
+### Pre Requisites
+
+* Docker
+* Postgres DB instance
+
+# Configuration
+
+- Container will use default values to connect Waltz to DB and try to `update` Postgres database and `run` Waltz.
+- You can change this with providing environment variables to container or as part of [docker-compose.yml](../docker-compose.yml)
+
+## Default values and actions
+ Container will execute two commands `update` and `run`. First command will `update` the database instance running `liquibase` command with default parameters for PostgreSQL instance listed below. Second command `run` will execute `catalina run` database and `run` Waltz.
+
+* `DB_HOST="postgres"`
+* `DB_PORT="5432"`
+* `DB_NAME="waltz"`
+* `DB_USER="waltz"`
+* `DB_PASSWORD="waltz"`
+* `DB_SCHEME="waltz"`
+* `WALTZ_FROM_EMAIL="help@finos.org"`
+* `WALTZ_BASE_URL="http://127.0.0.1:8080/"`
+* `CHANGELOG_FILE=_FILE="/opt/waltz/liquibase/db.changelog-master.xml"`
+
+# Running
+
+## Docker run
+
+Run waltz without updating database:
+
+    $> docker run -it ghcr.io/[OWNER]/[REPO]:postgresql run
```

Comment:
We should put the 'real' image in here so that people can just copy/paste to try out Waltz. We should also not use the `postgresql` tag here to be consistent with `docker-compose.yml`.

```suggestion
    $> docker run -it ghcr.io/finos/waltz run
```

Reply:
Also we should add some environment variables to show how to connect to an existing database instance (and use different values from the default to make that clear!)

Reply:
I believe that is shown few lines below:
```    
$> docker run -it ghcr.io/[OWNER]/[REPO]:postgresql \
      -e "DB_HOST=postgres" \
      -e "DB_NAME=waltz" \
      -e "DB_USER=waltz" \
      -e "DB_PASSWORD=waltz" \
      update run```

Reply:
Yes, but we should do it for both, otherwise it looks like you don't need any env var to run waltz without updating the database.
Also, we should use something else than the default values for those variables, otherwise it doesn't really make sense. If you were to run those commands as-is, they wouldn't work (unless there is a server called `postgres` somewhere on your network in the same domain as you).
Let me know if I'm not making sense! 😅

Reply:
yup, that sounds good! 👍 
maybe use `user` and `password` for the user and password — most important is that you use the same on both command lines 😄

Reply:
yep, make sense, got it all :D